remapping_pipeline.nf

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env nextflow
+
+def helpMessage() {
+    log.info"""
+    Index the provided genome with bowtie and samtools
+    Usage:
+            --oldgenome                     Fasta file containing the reference genome.
+            --outdir                    Directory where the index will be deployed.
+    """
+}
+params.flankingseq = 50
+params.scorecutoff = 0.6
+params.diffcutoff =  0.04
+params.outfile = "$baseDir/resources/remap_vcf.vcf"
+
+// Show help message
+if (params.help) exit 0, helpMessage()
+params.oldgenome = "$baseDir/resources/genome.fa"
+params.newgenome = "$baseDir/resources/genome.fa"
+params.vcffile = "$baseDir/resources/source_vcf.vcf"
+
+// Index files for both old and new genomes 
+oldgenome_chrom_sizes = file("${params.oldgenome}.chrom.sizes")
+oldgenome_fai = file("${params.oldgenome}.fai")
+newgenome_dir = file(params.newgenome).getParent()
+newgenome_basename = file(params.newgenome).getName()
+newgenome_fai = file("${params.newgenome}.fai")
```

Comment:
Some of the input files for this pipeline are actually the output files of the `genome_preparation.nf` step. As far as I understand, in the Nextflow world the recommended approach is write a single big pipeline inlcuding _all_ of the processing step — in this case, it would be easier to transfer files between steps without the need to define extra input and output files.

@tcezard @BaronKoy What do you think about the idea of joining the two `.nf` files into a single pipeline? Since this is a lot of work, I agree that this should be done as a separate ticket and not as part of this one.

Reply:
I think you're right. Especially since it is now quite easy to make processes run conditionally. 
I've created a new ticket to merge them back together.

Reply:
Linking issue here so it's easier to find — https://www.ebi.ac.uk/panda/jira/browse/EVA-2001